api/src/controllers/users.js

```diff
@@ -229,7 +229,7 @@ module.exports = {
   v2: {
     create: async (req, res) => {
       try {
-        await auth.check(req, 'can_create_users');
+        await auth.check(req, ['can_edit', 'can_create_users']);
```

Comment:
Change to `hasAllPermissions` (as above).

Reply:
`check` was refactored